Tests/resources/directory/TestController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Test;
+
+class TestController
+{
+    public function init()
+    {
+        $this->l('Shop');
+    }
+
+    public function productAction()
+    {
+        $this->trans('Fingers', [], 'admin.product.help');
```

Comment:
Best follow good practices, even if it would work at extractor level

```suggestion
        $this->trans('Fingers', [], 'Admin.Product.Help');
```